src/windows/guest_list.c

```diff
@@ -200,7 +200,7 @@ void window_guest_list_open()
 	window->enabled_widgets =
 		(1 << WIDX_CLOSE) |
 		(1 << WIDX_PAGE_DROPDOWN) |
-		(1 << WIDX_PAGE_DROPDOWN_BUTTON) |
+		//(1 << WIDX_PAGE_DROPDOWN_BUTTON) |
```

Comment:
Just noticed aswell. 65866365e333f530eb2693018409b33dd06612e5 is a messed up commit. will fix this today.